File: Baekjoon/S5/C#/S5_덩치.cs
https://www.acmicpc.net/problem/7568

using System;

namespace Baekjoon
{
    class Program
    {
        static void Main(string[] args)
        {
            int n = int.Parse(Console.ReadLine());
            int[] h = new int[n];
            int[] w = new int[n];

            for (int i = 0; i < n; i++)
            {
                int[] arr = Array.ConvertAll(Console.ReadLine().Split(' '), int.Parse);
                h[i] = arr[0];
                w[i] = arr[1];
            }

            for (int i = 0; i < n; i++)
            {
                int cnt = 1;
                for (int j = 0; j < n; j++)
                {
                    if (h[i] < h[j] && w[i] < w[j])
                        cnt++;
                }
                Console.WriteLine(cnt);
            }
        }
    }
}

File: Baekjoon/S5/JAVA/S5_덩치.java
//https://www.acmicpc.net/problem/7568

import java.io.*;
import java.util.*;

public class Main{
    public static void main(String args[]) throws IOException{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());
        
        int[][] xy = new int[n][2];
        int[] ans = new int[n];
        
        for(int i = 0; i < n; i++){
            String[] str = br.readLine().split(" ");
            xy[i][0] = Integer.parseInt(str[0]);
            xy[i][1] = Integer.parseInt(str[1]);
            ans[i] = 1;
        }
        
        StringBuilder sb = new StringBuilder();
        for(int i = 0; i < n; i++){
            for(int j = 0; j < n; j++){
                if(i != j){
                    if(xy[i][0] < xy[j][0] && xy[i][1] < xy[j][1])
                        ans[i]++;
                }
            }
            sb.append(ans[i]+"\n");
        }
        
        System.out.print(sb);
        br.close();
    }
}